### PR TITLE
Feature/RUN-5123 native window

### PR DESCRIPTION
**Pull request by rdepena**
#### Description of Change
Added the ability to get the [Window Object](https://developer.mozilla.org/en-US/docs/Web/API/Window) previously available in the V1 API as `getNativeWindow`

- [ X ] PR description included and stakeholders cc'd
- [ X ] `npm test` passes
- [ X ] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [] manual tests are [changed or added](https://github.com/openfin/test_apps)
- [X] relevant documentation is [changed or added](https://github.com/hadoukenio/js-adapter)
- [X] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [X] PR release notes describe the change in a way relevant to app-developers

Notes: 
Added `getWebWindow` API that returns the [Window Object](https://developer.mozilla.org/en-US/docs/Web/API/Window).  Previously `getNativeWindow` in API V1

Test Results: 
[Win 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5c9e79c1cb360141a7dfdf3a)
[Win 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5c9e7c0acb360141a7dfdf3b)

### CODE REVIEW

**Comment by rdepena** [comment]
updated tests:

[Win 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5ca21b39cb360141a7dfdf3e)
[Win 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5ca21c84cb360141a7dfdf3f)

File: src/renderer/api-decorator.js [hunk 1]
```diff
@@ -501,6 +502,41 @@
         return featuresObj;
     }
 
+    function registerWebWindow(name, win, from) {
+        webWindowMap.set(name, win);
```
Comment (licui3936) [inline src/renderer/api-decorator.js]: 'from' is not used.

File: src/renderer/api-decorator.js [hunk 2]
```diff
@@ -501,6 +502,41 @@
...
+    }
+
+    function getWebWindow(name) {
+        let webWindow = webWindowMap.get(name);
+        if (webWindow) {
+            return webWindow;
+        } else {
+            try {
+                return window.opener.fin.__internal_.getWebWindow(name);
+            } catch (err) {
```
Comment (licui3936) [inline src/renderer/api-decorator.js]: Question: If child window is cross domain, window.opener.fin will throw an exception. We're ok with it?

Comment (rdepena) [reply src/renderer/api-decorator.js]: yea, the exception is a good improvement over no feedback

File: src/renderer/api-decorator.js (hunk 1 above)
Comment (rdepena) [reply src/renderer/api-decorator.js]: good catch.

**Comment by rdepena** [comment]
@datamadic fixed scenario where parent reloads.

updated test run: 

[Win 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5ca4f866cb360141a7dfdf7f)
[Win 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5ca4f98dcb360141a7dfdf80)

**Pull request merged**